runtime/functions

```diff
@@ -260,7 +260,11 @@ trust_localnet() {
 
 set_resolvconf_perms() {
   echo "Setting resolv ACLs..."
-  setfacl -m user:${PG_USER}:r /etc/resolv.conf
+  if which setfacel >/dev/null; then
+    setfacl -m user:${PG_USER}:r /etc/resolv.conf
```

Comment:
This cannot be true. `setfacl` is installed in the image, its a part of the image

```bash
# docker run -it --rm sameersbn/postgresql setfacl
Usage: setfacl [-bkndRLP] { -m|-M|-x|-X ... } file ...
Try `setfacl --help' for more information.
```
Is it possible your using an old version of the image?